vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -34,6 +34,7 @@
 #include "bat/ads/internal/confirmations/confirmations.h"
 #include "bat/ads/internal/conversions/conversions.h"
 #include "bat/ads/internal/database/database_initialize.h"
+#include "bat/ads/internal/features/features.h"
```

Comment:
this should either be bat/ads/internal/features.h or bat/ads/internal/page_probabilities/features.h depending on how you want to organize things, but features/features.h is redundant and doesn't follow existing conventions

Reply:
moved one dir up to internal/*